ir/src/test/scala/org/scalajs/core/ir/PrintersTest.scala

```diff
@@ -918,22 +918,26 @@ class PrintersTest {
           |@hints(1) class LTest extends O {
           |}
         """,
-        ClassDef("LTest", ClassKind.Class, Some(ObjectClass), Nil, None, Nil)(
+        ClassDef("LTest", ClassKind.Class, Some(ObjectClass), Nil, None, Nil,
+            Nil)(
             NoOptHints.withInline(true)))
   }
 
   @Test def printClassDefDefs(): Unit = {
     assertPrintEquals(
         """
-          |class LTest extends O {
+          |module class LTest extends O {
           |  val x$1: int
           |  var y$1: int
+          |  export top module "pkg.Foo"
           |}
         """,
-        ClassDef("LTest", ClassKind.Class, Some(ObjectClass), Nil, None,
+        ClassDef("LTest", ClassKind.ModuleClass, Some(ObjectClass), Nil, None,
```

Comment:
Yes, because an `export top module` does not make sense in a non-module class. The IR would still correctly print, of course, but it would look weird.